src/common/schema/application.js

```diff
@@ -398,14 +398,14 @@ module.exports = {
     'dischargeType'
   ],
   dependencies: {
-    veteranGrossIncome: ['understandsFinancialDisclosure'],
-    veteranNetIncome: ['understandsFinancialDisclosure'],
-    veteranOtherIncome: ['understandsFinancialDisclosure'],
-    spouseGrossIncome: ['understandsFinancialDisclosure'],
-    spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure'],
-    deductibleMedicalExpenses: ['understandsFinancialDisclosure'],
-    deductibleFuneralExpenses: ['understandsFinancialDisclosure'],
-    deductibleEducationExpenses: ['understandsFinancialDisclosure']
+    veteranGrossIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    veteranNetIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    veteranOtherIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseGrossIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseNetIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
+    spouseOtherIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
```

Comment:
Yeah, I think we have the same view on this.  We don't really use jsonschema for conformance-- it's mostly there as a sanity check.  This seems like a reasonable amount of sanity to look for.

As to why the `spouse*` fields are behaving differently now, I'm still not sure.  `spouseFullName` isn't a required field, but it's behaving like it is.